.github/workflows/stale.yml

```diff
@@ -13,7 +13,9 @@ jobs:
     - uses: actions/stale@v3
       with:
         repo-token: ${{ secrets.RELEASE_USER_TOKEN }}
-        days-before-stale: 14
+        days-before-stale: 30
```

Comment:
The stale message is still 14 days - might also want updating on line 21

Reply:
Yep, good catch. :+1: